tests/validatingadmissionpolicy/noderestrictions.go

```diff
@@ -193,4 +206,69 @@ var _ = Describe("[sig-compute] virt-handler node restrictions via validatingAdm
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("%s should not fail on node specific node restriction", description))
 		}
 	})
+
+	Context("patching another node", func() {
+		BeforeEach(func() {
+			nodesList := libnode.GetAllSchedulableNodes(virtClient)
+			Expect(nodesList.Items).ToNot(BeEmpty())
+			Expect(nodesList.Items).ToNot(HaveLen(1))
+			for _, node := range nodesList.Items {
+				if nodeName != node.Name {
+					anotherNode = node.Name
+					break
+				}
+			}
+
+			prepareNode(anotherNode)
+		})
+
+		AfterEach(func() {
+			cleanup(anotherNode)
+		})
+
+		It("rejects kubevirt related patches", func() {
+			patchSetList := map[string]*patch.PatchSet{
+				"kubevirt.io label addition":      patch.New(patch.WithAdd("/metadata/labels/kubevirt.io~1newAllowedLabel", "value")),
+				"kubevirt.io label update":        patch.New(patch.WithReplace(allowedLabelPath, "other-value")),
+				"kubevirt.io label removal":       patch.New(patch.WithRemove(allowedLabelPath)),
+				"kubevirt.io annotation addition": patch.New(patch.WithAdd("/metadata/annotations/kubevirt.io~1newAllowedAnnotation", "value")),
+				"kubevirt.io annotation update":   patch.New(patch.WithReplace(allowedAnnotationPath, "other-value")),
+				"kubevirt.io annotation removal":  patch.New(patch.WithRemove(allowedAnnotationPath)),
+			}
```

Comment:
Why not make it a table?

Reply:
`DescribeTable` is to replace multiple `It`. This is imho one contained test.